cmds/deploy.go

```diff
@@ -418,9 +420,9 @@ func NewCmdDeploy(f *cmdutil.Factory) *cobra.Command {
 						// extract the ip address from the URL
 						ip := strings.Split(cfg.Host, ":")[1]
 						ip = strings.Replace(ip, "/", "", 2)
-						changed = addAnnotationIfNotExist(&node.ObjectMeta, "kubernetes.io/externalIP", ip)
+						changed = addAnnotationIfNotExist(&node.ObjectMeta, externalIPNodeLabel, ip)
```

Comment:
As discussed on IRC, this is unnecessary once the node IP is fixed in minikube/minishift.